### ISSUE TITLE
Config Copy Tool does not seem to work at all

### ISSUE THREAD

**Issue by TheNicolasDeveloper** [age-series/ElectricalAge#335]
When I Shift-Right Click a lamp, it does not copy its configuration at all.

Expected behaviour:
When I shift right click a lamp it should copy its configuration and I should be able to paste it into with a right click another lamp.

Mod Version: 1.22.3

**Comment by TheNicolasDeveloper** [comment]
ive took a look at the issue and i fixed it in a pull request i just made

**Comment by jrddunbr** [comment]
Yeah, this probably got broken when I refactored the key handler recently. I will have to test this and confirm it has the correct behavior.
